pulser/__init__.py

```diff
@@ -22,4 +22,4 @@
 
 from pulser.sequence import Sequence
 
-from pulser.simulation import Simulation
+from pulser.simulation import Simulation, SimConfig
```

Comment:
I would rather not have `SimConfig` importable from `pulser`, but rather from `pulser.simulation` only.